build_helpers/startSiteDevServer.sh

```diff
@@ -7,4 +7,4 @@ rm -rf ./__site_prerender__
 ./build_helpers/buildAPIDocs.sh
 webpack --config "$PWD/site/webpack-prerender.config.js"
 ./build_helpers/buildSiteIndexPages.sh
-webpack-dev-server --config "$PWD/site/webpack-client.config.js" --no-info --content-base __site__
+webpack-dev-server --config "$PWD/site/webpack-client.config.js" --no-info --content-base __site__ --port ${1-8080}
```

Comment:
Sweet.  Could you add a section to the Contributing.md doc similar to what was done for tests here: https://github.com/schrodinger/fixed-data-table-2/pull/167/files#diff-6a3371457528722a734f3c51d9238c13